Automattic/wp-calypso
# Gutenboarding: add new template designs

## Changes proposed in this Pull Request

⚠️ https://github.com/Automattic/wp-calypso/pull/42353 ~must be merged first~ merged

## Questions

1. What is the relationship between theme and design/template? For example, how do we distinguish between [Bowen](https://gutenboardinghp1.wordpress.com/) and [Coutoire](https://gutenboardinghp20.wordpress.com/)? 
2. I've been determining the fonts by looking at the heading and base fonts for the demo sites, e.g., https://coutoiredemo.wordpress.com/ Is this correct?

## TODO
- [x] Add the rest of the new themes/designs from #42353
- [ ] Get some indication on how to properly test this 
- [x] Add feature flag for testing? https://github.com/Automattic/wp-calypso/pull/42353#issuecomment-630672970
- [ ] Ask if any of the new designs should be "premium". See: https://github.com/Automattic/wp-calypso/pull/42377
- [x] Generate preview images: https://github.com/Automattic/wp-calypso/blob/master/bin/generate-gutenboarding-design-thumbnails.js (Re-generate after https://github.com/Automattic/wp-calypso/pull/43104 lands)

## Testing instructions

Head over to /new
Make sure the following designs feature in /new/design: [TO COME]
Ensure that the font selector and preview works for each design
Create a site using each design, checking that the resulting homepage matches the theme/design

Fixes #42208
